### PR TITLE
[MetaSchedule] Add "disabled_pass" option in tuning API

### PR DESCRIPTION
**Pull request by ibsidorenko**
Now there is no way to disable passes in MetaShedule tuner. This commit adds new parameter `disabled_pass` in tuning API (`tune_relay`/`compile_relay`). It can be used for different experiments and non default behavoir.

### CODE REVIEW

**Comment by ibsidorenko** [comment]
cc @masahi @junrushao

**Review by masahi — APPROVED** [review]

File: python/tvm/meta_schedule/relay_integration.py [hunk 1]
```diff
@@ -250,6 +252,7 @@ def tune_relay(
     seed: Optional[int] = None,
     module_equality: str = "structural",
     num_tuning_cores: Union[Literal["physical", "logical"], int] = "physical",
+    disabled_pass: Optional[Union[List[str], Set[str], Tuple[str]]] = None,
```
Comment (echuraev) [inline python/tvm/meta_schedule/relay_integration.py:255]: Could you please add information about this parameter to the docstring?

File: python/tvm/meta_schedule/relay_integration.py [hunk 2]
```diff
@@ -345,6 +350,7 @@ def compile_relay(
         }
     ),
     executor: Optional["relay.backend.Executor"] = None,
+    disabled_pass: Optional[Union[List[str], Set[str], Tuple[str]]] = None,
```
Comment (echuraev) [inline python/tvm/meta_schedule/relay_integration.py:353]: Add information to doc

**Review by echuraev — COMMENTED** [review]
Also, I have a question. Should we add a test for this option or not?

**Comment by ibsidorenko** [comment]
> Also, I have a question. Should we add a test for this option or not?

I thought about that... since this is rather trivial change, not a bug fixing + it can be checked with tuning only (time consuming operation) I decided not to add unit test.

File: python/tvm/meta_schedule/relay_integration.py (hunk 2 above)
Comment (ibsidorenko) [reply python/tvm/meta_schedule/relay_integration.py:357]: Done

File: python/tvm/meta_schedule/relay_integration.py (hunk 1 above)
Comment (ibsidorenko) [reply python/tvm/meta_schedule/relay_integration.py:257]: Done

**Review by echuraev — APPROVED** [review]
LGTM. Thanks

**Pull request merged**